setup.py

```diff
@@ -8,6 +8,9 @@
 import os
 import sys
 
+from fmriprep.info import (__version__, __author__, __copyright__, __credits__,
```

Comment:
Kk, I'll remove it. When I was attempting to do an install of fmriprep locally it was failing because it couldn't get at the information in fmriprep.info. I'll try and rework it to not import from a library that isn't installed yet.